src/core/friendly_errors/browser_errors.js

```diff
@@ -42,13 +52,73 @@ const strings = {
       msg: "expected {{.}}, got '{{.}}'",
       type: 'UNEXPECTEDTOKEN',
       browser: 'Chrome'
+    },
+    {
+      msg: "Identifier '{{.}}' has already been declared",
+      type: 'REDECLAREDVARIABLE',
+      browser: 'Chrome'
+    },
+    {
+      msg: 'redeclaration of {} {{.}}',
+      type: 'REDECLAREDVARIABLE',
+      browser: 'Firefox'
+    },
+    {
+      msg: 'Missing initializer in const declaration',
+      type: 'MISSINGINITIALIZER',
+      browser: 'Chrome'
+    },
+    {
+      msg: 'missing = in const declaration',
+      type: 'MISSINGINITIALIZER',
+      browser: 'Firefox'
+    },
+    {
+      msg: 'Illegal return statement',
+      type: 'BADRETURNORYIELD',
+      browser: 'Chrome'
+    },
+    {
+      msg: 'return not in function',
+      type: 'BADRETURNORYIELD',
+      browser: 'Firefox'
```

Comment:
Can we separate `BADRETURNORYIELD` and create `BADRETURN` & `BADYIELD`. Yield is not common for beginners and might be confusing to have both in the same error message

Reply:
How about having `BADRETURN` only because this error focuses on misplaced return statements. I named the type `BADRETURNORYIELD` because it matches the error mentioned in the MDN reference.

Reply:
Then maybe keep it as `BADRETURNORYIELD` and edit the text displayed so that it only shows return.

Reply:
I have made the changes please have a look